src/Entity/LREArea.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class LREArea
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @var ReferentTag|null
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\ReferentTag")
```

Comment:
do we really need to keep it nullable? if not, I propose to do: 
- `@JoinColumn(nullable=false)`
- `__construct(ReferentTag $tag)`